File: ITP1/3/d.cpp
#include <iostream>
using namespace std;

int main() {
    int a, b, c;
    cin >> a >> b >> c;
    int count = 0;
    for (int d=a;d<=b;d++) {
        if (c%d == 0) {
            count++;
        }
    }
    cout << count <<endl;
    return 0;
}

File: ITP1/3/d.js
let inputs = require('fs').readFileSync('/dev/stdin', 'utf8').split(" ");

const a = ~~inputs[0];
const b = ~~inputs[1];
const c = ~~inputs[2];

let count = 0;
for (d=a;d<=b;d++) {
    if (c%d === 0) {
        count += 1;
    };
};
console.log(count);
